package.json

```diff
@@ -31,8 +31,8 @@
     "popper.js": "^1.12",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
-    "axios": "^0.18",
-    "bootstrap": "^4.0.0"
+    "axios": "^0.19",
+    "bootstrap": "^4.1.0"
```

Comment:
Would you say it's worth updating this to `4.3`?

Reply:
This is just an update that matches how this configured in the base Larvel 5.8 framework. Because of semantic versioning, npm should still install 4.3 regardless.